documentation/asciidoc/topics/attributes/downstream-attributes.adoc

```diff
@@ -13,5 +13,5 @@
 
 :doc_home: https://access.redhat.com/documentation/en-us/red_hat_data_grid/
 :download_url: https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?product=data.grid&downloadType=distributions
-:cli_docs: https://access.redhat.com/documentation/en-us/red_hat_data_grid/8.1/html-single/data_grid_command_line_interface/index
-:cpp_api_docs: https://access.redhat.com/webassets/avalon/d/red-hat-data-grid/8.1/cpp/index.html
+:cli_docs: https://access.redhat.com/documentation/en-us/red_hat_data_grid/8.4/html-single/data_grid_command_line_interface/index
```

Comment:
seems it's changed to:
```suggestion
:cli_docs:https://access.redhat.com/documentation/en-us/red_hat_data_grid/8.4/html-single/using_the_data_grid_command_line_interface/index
```

Reply:
Yes you're right. Thank you for checking the url